packages/manager/modules/carbon-calculator/src/dashboard/routing.js

```diff
@@ -9,6 +9,11 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.href($state.current.name, $transition$.params()),
       dashboardLink: /* @ngInject */ ($state) => $state.href('app.dashboard'),
       breadcrumb: () => null,
+      hasInvoice: /* @ngInject */ ($http) =>
+        $http
+          .get('/me/carbonCalculator/hasInvoice')
+          .then(({ data }) => data.hasInvoice)
+          .catch(() => false),
```

Comment:
Should be moved into `CarbonFootprintService` to gather API calls. 

```suggestion
      hasInvoice: /* @ngInject */ (CarbonFootprintService) =>
       CarbonFootprintService.hasInvoice(),
```